src/app/lib/globalStyles/index.jsx

```diff
@@ -22,9 +23,17 @@ const GlobalStyleComponent = createGlobalStyle`
     ${({ fonts }) => fonts && createFontStyles(fonts)}
   `;
 
-const GlobalStyle = () => {
-  const { fonts } = useContext(ServiceContext);
-  return <GlobalStyleComponent fonts={fonts} />;
+const GlobalStyle = ({ fonts }) => {
+  const { fonts: styleFonts } = useContext(ServiceContext);
+  return <GlobalStyleComponent fonts={fonts || styleFonts} />;
+};
+
+GlobalStyle.propTypes = {
+  fonts: string,
+};
+
+GlobalStyle.defaultProps = {
```

Comment:
@jimjohnsonrollings added this default props to ReithSans and just wanted be sure that it should be the default.
cc: @jamesdonoh @rhell

Reply:
@RayNjeri what effect would this have?  Several services will not want to use Reith fonts at all.

Reply:
Yeah i got that, since we are not sure what we want to be the default i opted to turn the Eslint rule on it..